src/Command/MigrateEpicsCommand.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Epic;
+use App\Entity\Issue;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'app:migrate-epics',
+    description: 'Migrate epics for existing legacy issues, create the epics and add link to join table.',
+)]
+class MigrateEpicsCommand extends Command
+{
+    protected static $defaultName = 'app:assign-epics'; // Default name of the command
+    protected static $defaultDescription = 'Assigns Epics to Issues based on the epicName property.';
+
+    private EntityManagerInterface $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        parent::__construct();
+        $this->entityManager = $entityManager;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $issueRepository = $this->entityManager->getRepository(Issue::class);
+        $epicRepository = $this->entityManager->getRepository(Epic::class);
+
+        // Get all issues
+        $issues = $issueRepository->findAll();
+        if (!$issues) {
```

Comment:
findAll returns an array...
```suggestion
        if (empty($issues)) {
```